ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx

```diff
@@ -79,7 +81,7 @@ export default function DiagnosisTable({ diagnosis }: Props) {
           {c}
         </h1>
       ))}
-      <h3 className="is-size-4">{Title}</h3>
+      <h3 className="is-size-4">{t(title)}</h3>
```

Comment:
```
t(`diagnosis.tables.${Title}`)
```

Reply:
Got it, i will do this change for Title in next pr which would add I18N support for `Category/Comment` of diagnosis table.